docs/configuration-frontend.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[configuration-frontend]]
+== Set up Frontend support
+
+At the moment the {apm-rum-ref}/index.html[JavaScript RUM Agent] 
+offers frontend support by supporting _real user monitoring (rum)_.
+
+Example config with frontent support enabled:
+
+["source","yaml"]
+----
+apm-server.frontend.enabled: true 
+apm-server.frontend.rate_limit: 10 
+apm-server.frontend.allow_origins: ['*'] 
+apm-server.frontend.library_pattern: "node_modules|bower_components|~" 
+apm-server.frontend.exclude_from_grouping: "^/webpack" 
+apm-server.frontend.source_mapping.cache.expiration: 5m 
+apm-server.frontend.source_mapping.index_pattern: "apm-*-sourcemap*" 
+----
+
+[float]
+=== Configuration options
+
+[[frontend-enable]]
+[float]
+==== `enabled` 
+For enabling frontend support, set the `apm-server.frontend.enabled` to `true`.
+By default this is disabled.
+
+[float]
+==== `rate_limit`
+Rate limit per second and IP address for requests sent to the frontend endpoint.
```

Comment:
worth mentioning that can't be disabled? (or to set a very high number to pretend to be disabled)